performance-timeline/po-observe.html

```diff
@@ -40,9 +40,29 @@
   }, "Filter unsupported entryType entryType names within the entryTypes sequence");
 
   async_test(function (t) {
-  var observer = new PerformanceObserver(
+    var finish = t.step_func(function () { t.done(); });
+    var observer = new PerformanceObserver(  
+      function (entryList, obs) {
+        var self = this;
```

Comment:
I filed https://github.com/w3c/testharness.js/issues/236 to make this better. Once that is resolved we can use a less ugly approach here.